bundles/org.openhab.binding.proteusecometer/README.md

```diff
@@ -0,0 +1,134 @@
+# ProteusEcoMeter Binding
+
+This is the binding for the Proteus EcoMeter S, which is able to report the level of a cistern or tank.
+
+Note that this binding currently supports no write channels. This means you have to configure your sensor by
+considering the manual of the product (using wireless display). After doing that the binding comes into play
+and helps you to get your measured values into openHAB. Please be patient while waiting for the first received
+data. The sensor reports at an interval of approx. 1h, except when the water level changes relatively fast.
```

Comment:
```suggestion
Note that this binding currently supports no write channels.
This means you have to configure your sensor by considering the manual of the product (using wireless display).
After doing that the binding comes into play and helps you to get your measured values into openHAB.
Please be patient while waiting for the first received data.
The sensor reports at an interval of approx. 1h, except when the water level changes relatively fast.
```

Please do this for the rest of the readme. Each sentence on its own line so it is cleaner if a change is needed to be made and reviewed.

If you accept this change or any others, don't forget to add your signoff on the last line on the comment field.